app.json

```diff
@@ -9,7 +9,6 @@
     "platforms": ["ios", "android"],
     "version": "3.1.2",
     "runtimeVersion": "nativeVersion",
-    "orientation": "portrait",
```

Comment:
why was this removed?

https://docs.expo.dev/versions/latest/config/app/#orientation

Reply:
since the `initialOrientation` value is set to `DEFAULT` in the plugin section, we do not need the `orientation` value here. if we set the `orientation` value to `portrait`, we cannot switch the application to landscape mode in our builds for iOS.

Reply:
yes you are right the lock has been removed in `OnboardingManager`. but why don't we want the application to be rotated at startup?

Reply:
i think it was related to splash screen images being bad at landscape mode. maybe you find some git history entry about it from the past?

Reply:
I have tried different builds and now I think that we don't need to set `initialOrientation` to `DEFAULT`.
It is enough to add `expo-screen-orientation` as a plugin.
Initially the application is locked in portrait mode, but afterwards it can be switched to landscape mode when locked to `DEFAULT` mode.

Reply:
i built new development apps locally for android and ios.
on android, the orientation works fine.
sadly i have a lot of the following warnings, which i did not knew from earlier versions:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/1942953/232451451-9481614f-3b94-436c-a13e-b657c30641e6.png">

on ios, i cannot change orientation in the dev app. when i do an upside down, i see that the status bar goes away, but the screen stays.

any idea, why?

Reply:
I don't know the cause of the image source null error. This is the first time I've seen it.

---
I'm sorry to hear that. It worked fine for me. I'll try it and let you know more.

Reply:
After my recent tests, I think this is the code that makes Orientation work the most correctly.

<img width="286" alt="image" src="https://user-images.githubusercontent.com/11755668/233052062-660dec38-d69e-4f92-8ee4-801c25c16302.png">

The application initially opens in Portrait mode and can then be switched to Landscape mode.

|landscape|portrait|
|--|--|
![Simulator Screen Shot - iPhone 14 Plus - 2023-04-19 at 13 04 57](https://user-images.githubusercontent.com/11755668/233055969-ea32ba3f-d5c6-4313-a283-ee91e77ef232.png) | ![Simulator Screen Shot - iPhone 14 Plus - 2023-04-19 at 13 04 59](https://user-images.githubusercontent.com/11755668/233056019-a1420c23-34b2-44ff-8f51-07fbb13e57b6.png)